commit b2c9de21812b030103d11d6cf62ba7895441885d
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Wed Nov 18 09:30:36 2015 +0200

    ethdev: fix missing symbol export for rte_eth_dma_zone_reserve()
    
    Fixes: 719dbebceb81 ("xen: allow determining DOM0 at runtime")
    
    Signed-off-by: Panu Matilainen <pmatilai@redhat.com>

diff --git a/lib/librte_ether/rte_ether_version.map b/lib/librte_ether/rte_ether_version.map
index 11c4e94..0c35d53 100644
--- a/lib/librte_ether/rte_ether_version.map
+++ b/lib/librte_ether/rte_ether_version.map
@@ -133,6 +133,7 @@ DPDK_2.2 {
 
 	rte_eth_copy_pci_info;
 	rte_eth_dev_get_dcb_info;
+	rte_eth_dma_zone_reserve;
 	rte_eth_rx_queue_info_get;
 	rte_eth_timesync_adjust_time;
 	rte_eth_timesync_read_time;
